force-app/main/default/lwc/serviceScheduleAddSessionsQuickAction/serviceScheduleAddSessionsQuickAction.js

```diff
@@ -0,0 +1,18 @@
+/*
+ *
+ *  * Copyright (c) 2022, salesforce.com, inc.
+ *  * All rights reserved.
+ *  * SPDX-License-Identifier: BSD-3-Clause
+ *  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ *
+ */
+
+import { LightningElement, api } from "lwc";
+import addServiceSessions from "@salesforce/label/c.Add_More_Sessions";
+
+export default class ServiceScheduleAddSessionsQuickAction extends LightningElement {
+    @api recordId;
+    @api isCommunity;
```

Comment:
This is never set, thinking it is not needed. I do not believe we need to do anything special with quick actions in a community page. At least it didn't look like it on the other quick actions. Side note this is not showing up in the community at all right now.